photo-portfolio/src/main/java/com/throughapinhole/photoportfolio/Classes/Post.java

```diff
@@ -0,0 +1,81 @@
+package com.throughapinhole.photoportfolio.Classes;
+
+import java.util.ArrayList;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter @Setter @NoArgsConstructor
+/**
+ * Represents a Post of the CMS
+ */
+public class Post {
+    private int id;
+    /** Date/Time Post was last modified */
+    private String modified;
+    /** Type of Post */
+    private String type;
+    /** Text title of Post */
+    private String title;
+    /** Text content of Post */
+    private String content;
+    /** User ID of author of Post */
+    private int author;
+    /** Media ID of featured Media of Post */
+    private int featuredMedia;
+    /** List of Category IDs assigned to the Post */
+    private ArrayList<Integer> categories;
```

Comment:
This should probably be `categoryIds` right?

Reply:
Changed all these namings to indicate they are Ids